modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp

```diff
@@ -537,6 +537,24 @@ renderResponse.setTitle(headerTitle);
 						/>
 					</aui:fieldset>
 
+					<c:if test="<%= FFExpirationDateReviewDateConfigurationUtil.expirationDateEnabled() || FFExpirationDateReviewDateConfigurationUtil.reviewDateEnabled() %>">
+						<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" label="expiration-date">
+							<c:if test="<%= FFExpirationDateReviewDateConfigurationUtil.expirationDateEnabled() %>">
+								<liferay-ui:error exception="<%= DLExpirationDateException.class %>" message="please-enter-a-valid-expiration-date" />
+
+								<p class="mb-5 text-secondary">
```

Comment:
We don't need for margin-button with `mb-5` without this CSS class works with 1 rem of margin like in categorization.

![image](https://user-images.githubusercontent.com/67901/119482102-c29df880-bd53-11eb-9735-6a8a16b58bdb.png)